sklearn/datasets/lfw.py

```diff
@@ -146,7 +146,7 @@ def _load_imgs(file_paths, slice_, color, resize):
     if not color:
         faces = np.zeros((n_faces, h, w), dtype=np.float32)
     else:
-        faces = np.zeros((n_faces, h, w, 3), dtype=np.float32)
+        faces = np.zeros((n_faces, int(h), int(w), 3), dtype=np.float32)
```

Comment:
I just don't understand where they would become float, but maybe i'm overlooking something.
We shouldn't add casts because we don't understand whats happening. If there is an operation that makes h and w floats (which they shouldn't be) then we should add the cast there. Lines 136 and 137 are integer divisions, right? Well we could definitely make them ``//`` (I'm assuming python2.7 here, btw. Did you get the warning in python3x? For python3 it should definitely be ``//``